qa/rpc-tests/test_framework/electrumutil.py

```diff
@@ -14,6 +20,56 @@ def compare(node, key, expected, is_debug_data = False):
 
 def bitcoind_electrum_args():
     import random
+    global ELECTRUM_PORT
+    ELECTRUM_PORT = random.randint(40000, 60000)
     return ["-electrum=1", "-debug=electrum", "-debug=rpc",
-            "-electrum.port=" + str(random.randint(40000, 60000)),
+            "-electrum.port=" + str(ELECTRUM_PORT),
             "-electrum.monitoring.port=" + str(random.randint(40000, 60000))]
+
+class ElectrumConnection:
+    def __init__(self):
+        self.s = socket.create_connection(("127.0.0.1", ELECTRUM_PORT))
+        self.f = self.s.makefile('r')
+        self.id = 0
+
+    def call(self, method, *args):
+        req = {
+            'id': self.id,
+            'method': method,
+            'params': list(args),
+        }
+        msg = json.dumps(req) + '\n'
+        self.s.sendall(msg.encode('ascii'))
+        res = self.f.readline()
+        return json.loads(res)
+
+# Helper function to attempt several times to connect to electrum server.
+# At startup, it may take a while before the server accepts connections.
```

Comment:
I don't mind much and this could be definitely called a nit, but shouldn't we use the same style for comment? I mean in `cashaddress.py` we used the py way to document methods, whereas here we are putting it prev to method definition.

Reply:
I don't know

Reply:
yes, use a docstring (string after the def) so that the function description is accessible programatically.